templates/CRM/Contribute/Form/Task/Invoice.hlp

```diff
@@ -32,3 +32,7 @@
 <p>{ts}You can send your email as a simple text-only message, as an HTML formatted message, or both. Text-only messages are sufficient for most email communication, and some recipients may prefer not to receive HTML formatted messages.{/ts}</p>
 <p>{ts}HTML messages have more visual impact, allow you to include images, and may be more readable if you are including links to website pages. However, different email programs may interpret HTML formats differently, so use this option cautiously unless you have a template format that has been tested with different web and desktop email programs.{/ts}</p>
 {/htxt}
+
+{htxt id="id-email_comment"}
+{ts}If you would like to add personal message to email please add it here. (If sending to more then one receipient the same message will be sent to each contact.){/ts}
```

Comment:
Not a blocker since this is the same as the original, but there's two typos and one grammar wrong it is.

But otherwise looks good.

Reply:
That is pretty bad, have replaced with new text. I closed this by accident and don't think I have the power to re-open - but can you?

Also the email message thing doesn't work at all, but I'll fix that in a new PR.

Reply:
Apparently I do.

And yeah I noticed the additional message didn't seem to do anything.

Reply:
Now it [will work](https://github.com/civicrm/civicrm-core/pull/26534)